scripts/template_openxr_dispatchdynamic.hpp

```diff
@@ -44,8 +42,7 @@ namespace OPENXR_HPP_NAMESPACE {
  * This is safer to use, especially in large/multi-module applications, than DispatchLoaderStatic, and is thus recommended.
  *
  * By default, it is lazy-populating: only populating a function pointer when it is attempted to be called (if this object is not
- * const). You can early-populate it using the createFullyPopulated() factory method, providing an Instance and optionally a
- * xrGetInstanceProcAddr function pointer.
+ * const). You can early-populate it using the populateFully() method, providing an Instance.
```

Comment:
Is there a specific reason for this API change?

Reply:
createFullyPopulated() is not a static function anymore because we sometimes want to fill it in two passes (with populateBase first). I think I was wrong to remove the optional xrGetInstanceProcAddr function pointer so I should add it again.